posthog/api/instance_settings.py

```diff
@@ -37,13 +38,16 @@ def get_instance_setting(key: str, setting_config: Dict = {}) -> InstanceSetting
         for _key, setting_config in settings.CONFIG.items():
             if _key == key:
                 break
+    is_secret = key in SECRET_SETTINGS
+    value = getattr(config, key)
 
     return InstanceSetting(
         key=key,
-        value=getattr(config, key),
+        value=value if not is_secret or not value else "*****",
```

Comment:
Why return the asterisks if there already is a more explicit field `is_secret`?

Reply:
Well I had to return something and `None` or `""` would introduce confusion on whether the value was set. Do you suggest another thing to return?